pkgs/applications/editors/vim/plugins/nvim-treesitter/generated.nix

```diff
@@ -414,12 +414,12 @@
   };
   diff = buildGrammar {
     language = "diff";
-    version = "0.0.0+rev=710e881";
```

Comment:
How did you generate this?

Reply:
I just ran the update script 🤷

Reply:
It's because the way versions are formatted are weird. It's something to look closely after the branch-off tho.